### ISSUE TITLE
FID evaluation

### ISSUE THREAD

**Issue by kkgoer** [YonghaoXu/Txt2Img-MHN#8]
Hi, can you provide pre-trained Inception model for  Inception score and FID score evaluation? I look forward to your reply, thanks!

**Comment by YonghaoXu** [comment]
Hi, I didn't back up this model. You can easily train it by running `pretrain_inception.py` in the `is_fid_score` folder.

**Comment by kkgoer** [comment]
> Hi, I didn't back up this model. You can easily train it by running `pretrain_inception.py` in the `is_fid_score` folder.

Thank you for your reply, one more small question, was the data without categories in the dataset not used to train the INCEPTION model, e.g. 00001.jpg?

**Comment by YonghaoXu** [comment]
Yes, those images are not considered for training the Inception model as the class information may be unclear.

**Comment by kkgoer** [comment]
> Yes, those images are not considered for training the Inception model as the class information may be unclear.

Thank you for your reply, and one last small question, the GAN-based methods you compared in your paper, when you tested them, did you pick the last epoch saved model?

**Comment by YonghaoXu** [comment]
Yes, the model at the last epoch after training was used for evaluation.
